tinyinstinstrumentation.cpp

```diff
@@ -136,6 +135,117 @@ RunResult TinyInstInstrumentation::RunWithCrashAnalysis(int argc, char** argv, u
   return ret;
 }
 
+RunResult TinyInstInstrumentation::Attach(char * script, char * process_name, uint32_t init_timeout, uint32_t timeout) {

```

Comment:
This function duplicates a lot of code from `TinyInstInstrumentation::Run`. Would be better if this was implemented by changing `TinyInstInstrumentation::Run` where appropriate. Probably the same thing with `TinyInstInstrumentation::AttachWithCrashAnalysis` and `TinyInstInstrumentation::RunWithCrashAnalysis`.